packages/stark-core/package.json

```diff
@@ -1,8 +1,12 @@
 {
   "name": "@nationalbankbelgium/stark-core",
   "version": "0.0.0-PLACEHOLDER-VERSION",
-  "module": "./esm5/stark-core.js",
-  "es2015": "./es2015/stark-core.js",
+  "module": "./fesm5/stark-core.js",
+  "es2015": "./fesm2015/stark-core.js",
+  "esm5": "./esm5/stark-core.js",
+  "esm2015": "./esm2015/stark-core.js",
+  "fesm5": "./fesm5/stark-core.js",
+  "fesm2015": "fesm2015/stark-core.js",
```

Comment:
Ok, so I think this is the change related to APF v6 right?

One thing, the path of the fesm2015 doesn't have './'like the others

Reply:
Indeed, APF v6.
And there's a mistake indeed, there should be a ./ in front (damn ^^)